data/org.freefilesync.FreeFileSync.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.freefilesync.FreeFileSync</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>FreeFileSync</name>
+  <summary>Visual folder comparison and synchronization</summary>
+  <description>
+    <p>
+      FreeFileSync is a folder comparison and synchronization software that
+      creates and manages backup copies of your important files. Instead of
+      copying every file every time, FreeFileSync determines the differences
+      between a source and a target folder and transfers only the minimum
+      amount of data needed.
+    </p>
+  </description>
+  <url type="homepage">https://www.freefilesync.org</url>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Manual synchronization of two folders</caption>
+      <image>https://www.freefilesync.org/images/screenshots/openSUSE.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Online storage support</caption>
+      <image>https://www.freefilesync.org/images/screenshots/sftp.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Synchronization settings</caption>
+      <image>https://www.freefilesync.org/images/screenshots/sync-settings.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Automatic synchronization</caption>
+      <image>https://www.freefilesync.org/images/screenshots/RealTimeSync.png</image>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">org.freefilesync.FreeFileSync.desktop</launchable>
+  <launchable type="desktop-id">org.freefilesync.FreeFileSync.RealTimeSync.desktop</launchable>
+  <provides>
+    <binary>FreeFileSync</binary>
+    <binary>RealTimeSync</binary>
+  </provides>
+  <releases>
+    <release version="9.5" date="2017-11-05"/>
+  </releases>
```

Comment:
This is a nicely done appdata file! If possible could you add [OARS](https://odrs.gnome.org/oars) metadata as well? Then it'd be full marks!

Reply:
I looked at it and it's clearly targeted at games. It doesn't even take something like software applications into account. Does it really make sense to try to shoehorn this into rating software?